client/ayon_core/hosts/max/plugins/publish/validate_mesh_has_uv.py

```diff
@@ -0,0 +1,57 @@
+
+import pyblish.api
+from ayon_core.hosts.max.api.action import SelectInvalidAction
+from ayon_core.pipeline.publish import (
+    ValidateMeshOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from pymxs import runtime as rt
+
+
+class ValidateMeshHasUVs(pyblish.api.InstancePlugin,
+                         OptionalPyblishPluginMixin):
+
+    """Validate the current mesh has UVs.
+
+    This validator only checks if the mesh has UVs but not
+    whether all the individual faces of the mesh have UVs.
+
+    It validates whether the current mesh has texture vertices.
+    If the mesh does not have texture vertices, it does not
+    have UVs in Max.
+
+    """
+
+    order = ValidateMeshOrder
+    hosts = ['max']
+    families = ['model']
+    label = 'Validate Mesh Has UVs'
+    actions = [SelectInvalidAction]
+    optional = True
+
+
```

Comment:
```suggestion
```
Still think the hound is disabled on this repo? @jakubjezek001 
This should've been caught?

Reply:
There's still a enter too many here. Right?